plugins/cert-manager.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cert-manager
+spec:
+  version: v0.16.0-alpha.0
+  homepage: https://github.com/jetstack/cert-manager
+  shortDescription: Manage cert-manager resources inside your cluster
+  description: |
+    A plugin accompanying cert-manger, a Kubernetes add-on to automate the management
```

Comment:
Since this is the official plugin for the project, you could also mention this, like

> The official plugin accompanying...